config/drupal-10/drupal-10.2-deprecations.php

```diff
@@ -3,12 +3,27 @@
 declare(strict_types=1);
 
 use DrupalRector\Rector\Deprecation\FunctionToStaticRector;
+use DrupalRector\Rector\Deprecation\MethodToMethodWithCheckRector;
 use DrupalRector\Rector\ValueObject\FunctionToStaticConfiguration;
+use DrupalRector\Rector\ValueObject\MethodToMethodWithCheckConfiguration;
 use Rector\Config\RectorConfig;
+use Rector\Renaming\Rector\PropertyFetch\RenamePropertyRector;
+use Rector\Renaming\ValueObject\RenameProperty;
 
 return static function (RectorConfig $rectorConfig): void {
     // https://www.drupal.org/node/2999981
     $rectorConfig->ruleWithConfiguration(FunctionToStaticRector::class, [
         new FunctionToStaticConfiguration('10.2.0', 'format_size', '\Drupal\Core\StringTranslation\ByteSizeMarkup', 'create'),
     ]);
+
+    // https://www.drupal.org/node/3265963
+    $rectorConfig->ruleWithConfiguration(MethodToMethodWithCheckRector::class, [
+        new MethodToMethodWithCheckConfiguration('Drupal\system\Plugin\ImageToolkit\GDToolkit', 'getResource', 'getImage'),
+        new MethodToMethodWithCheckConfiguration('Drupal\system\Plugin\ImageToolkit\GDToolkit', 'setResource', 'setImage'),
+    ]);
+
+    // https://www.drupal.org/node/3265963
+    $rectorConfig->ruleWithConfiguration(RenamePropertyRector::class, [
```

Comment:
Unfortunately this does not support backwards compatible fixes. We either need to pull that code into a class that extends our base class, or perhaps think up a wrapper that can wrap external Rector classes. Not sure how feasable that is though. 

I can imagine something like a class that allows for a reference to the rector as a configuration object, this could then be used to perhaps get a node types. This might break though if you try and wrap multiple (in config) since those may have different node types they apply to, resulting in breakage.